packages/translations/src/i18next/i18next.ts

```diff
@@ -133,7 +133,7 @@ export const changeLanguage = async (lang: string, cb: (arg0: string) => void) =
 // <Localize /> component wrapped with i18n
 export const Localize = withI18n(i18n);
 
-export const localize = <T extends object>(string: string, values?: T) => {
+export const localize = <T extends object>(string: string, values?: T): string => {
```

Comment:
a little help to @farzin-deriv to make TS better 😉 When added as const to our constants it couldn't infer type from localize()